sites/partners/src/components/applications/PaperApplicationForm/PaperApplicationForm.tsx

```diff
@@ -147,11 +142,11 @@ const ApplicationForm = ({ listingId, editMode, application }: ApplicationFormPr
       setLoading(false)
 
       if (result) {
-        setSiteAlertMessage(
+        addToast(
           editMode
             ? t("application.add.applicationUpdated")
             : t("application.add.applicationSubmitted"),
-          "success"
+          { variant: "success" }
```

Comment:
I think it would be nice to have a frontend enum for all the variant types. Not a blocking request though

Reply:
@mcgarrye so about that…I forget the exact reasoning but in the new Seeds components we dropped explicit TS enum types and just went with string unions inside the props interface. I'll double-check this PR though as I know we can import individual prop types if we need to for Core usage, and not need to re-type "success" | "warning" | etc.

Reply:
Yeah I double-checked — we're pulling in the variant terms directly from the TS interface, so it should work for autocomplete/type-checking.